pymoab-visit-py2-18.04

```diff
@@ -0,0 +1,28 @@
+FROM svalinn/pymoab-py2-18.04:latest
+
+ENV HOME /root
+
+RUN apt-get install -y --fix-missing \
```

Comment:
I would add a apt-get update at the beginning of this line, to avoid relying of this being done in the `svalinn/pymoab-py2-18.04:latest` container